packages/modular-scripts/src/utils/actionPreflightCheck.ts

```diff
@@ -1,11 +1,14 @@
 import * as logger from './logger';
+import getModularRoot from './getModularRoot';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ModularAction = (...args: any[]) => Promise<void>;
 
 function actionPreflightCheck(fn: ModularAction): ModularAction {
   const wrappedFn: ModularAction = async (...args) => {
     if (process.env.SKIP_PREFLIGHT_CHECK !== 'true') {
+      // Bail out if there is no modular root
+      getModularRoot();
```

Comment:
I would put this above the `SKIP_PREFLIGHT_CHECK` so that it always run because otherwise we could get into a situation where people just confused as to what's going go on. The preflight check is really about whether an "actual" modular repo is structurally correct. However we *always* want to make sure we're in a modular repo first.

Reply:
@LukeSheard - done